index.php

```diff
@@ -21,64 +21,75 @@
-                    
-                    <br> Działania w trakcie podanych semestrów: <br>
-                    
-                    <input class = "wideField" type = text name = "achievement[0][name]">
-                    <input type = date name = "achievement[0][startDate]">
-                    - <input type = date name = "achievement[0][endDate]"><br>
-                    
-                    <input class = "wideField" type = text name = "achievement[1][name]">
-                    <input type = date name = "achievement[1][startDate]">
-                    - <input type = date name = "achievement[1][endDate]"><br>
-                    
-                    <input class = "wideField" type = text name = "achievement[2][name]">
-                    <input type = date name = "achievement[2][startDate]">
-                    - <input type = date name = "achievement[2][endDate]"><br>
-                    <hr>                    
-                    
-                    <input type="submit" value = "Wygeneruj" />
-                         </form></div>
-
-         <hr>
-    </body>
+
+    <div class="form-container">
+
+        <form class="form" action="index.php" method="post">
+            <label>Imię i Nazwisko: <br /><input type="text" name="name" /></label>
+            <label>Indeks: <br /><input type="number" name="index" /></label>
+            <label>Funkcja: <br /><input type="text" name="function" /></label>
+            <label>Przewodniczący: <br /><input type="text" name="leader" /></label>
+
+            <label class="semesters">Semestry członkostwa: <br />
+                <input type="text" name="semester[]" placeholder="np.: zimowy 2019/2020" /><br />
+                <input type="text" name="semester[]" /><br /></label>
+
+            <p>Działania w trakcie podanych semestrów: </p>
+
+            <div class="achievement-container">
+                <div class="achievement-field">
+                    <textarea class="wideField" name="achievement[0][name]" cols="30" rows="5"></textarea>
+                    <div class="achievement-field-dates"><input type=date name="achievement[0][startDate]">
+                        <span> - </span> <input type=date name="achievement[0][endDate]"></div>
+                </div>
+            </div>
+
+            <span class="add-achievement-button">Dodaj osiągnięcie</span>
+
+            <input type="submit" value="Wygeneruj" />
+        </form>
+    </div>
+
+    <footer class="footer">Created by <a href="https://akai.org.pl" target="blank"><svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="189.211px" height="211.778px" viewBox="0 0 189.211 211.778" enable-background="new 0 0 189.211 211.778" xml:space="preserve">
+                <g>
+                    <path fill-rule="evenodd" clip-rule="evenodd" fill="#FAA21B" d="M45.2,21.895c24.751,0,44.853,19.896,45.192,44.566H90.4v0.634
```

Comment:
Wstawianie SVG bezpośrednio trochę zamula widok, może przenieść to do osobnego pliku? Obstawiam, że nie zagłębiałeś się w system budowania assetów więc na razie merge'uję tak jak jest z następnym taskiem - wstawienia jakiegoś systemu buildów ;)